### PR TITLE
update readme

### LINKED ISSUE #326: Typo In README for Core Package Import
The README has this line:
`from zowe.zos_core_for_zowe_sdk import ProfileManager`

However, it looks like it should be:
`from zowe.core_for_zowe_sdk import ProfileManager`

### LINKED ISSUE #328: Consistent Name for SDK
In the README the SDK is referred to as:
* Zowe Client Python SDK
* Zowe Python Client SDK

The name of the SDK should be consistently used and named across SDKs.  Related to https://github.com/zowe/docs-site/issues/3851

### PR DESCRIPTION
**Pull request by olamidepeterojo**
**What It Does**
Updated typo in README
Consistent use of SDK name across SDKs

Fixes #326 
Fixes #328
**How to Test**

**Review Checklist**
I certify that I have:
- [ ] tested my changes
- [ ] added/updated automated tests
- [x] updated the changelog
- [x] followed the [contribution guidelines](https://github.com/zowe/zowe-cli/blob/master/CONTRIBUTING.md)

**Additional Comments**

### CODE REVIEW

**Comment by zFernand0** [comment]
closing in favor of #341 (since it contains the same changes)

**Review by zFernand0 — CHANGES_REQUESTED** [review]
Hey @olamidepeterojo,
Please make sure to signoff your commits 😋 

To add your Signed-off-by line to every commit in this branch:

1. Ensure you have a local copy of your branch by [checking out the pull request locally via command line](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/checking-out-pull-requests-locally).
2. In your local branch, run: `git rebase HEAD~1 --signoff`
3. Force push your changes to overwrite the branch: `git push --force-with-lease origin readme-update`

**Comment by olamidepeterojo** [comment]
Thank you for the kind gesture and efforts in going through my PR... Will do accordingly

**Review by zFernand0 — APPROVED** [review]
LGTM! 😋

**Comment by olamidepeterojo** [comment]
Added the fix for this issue #328 to this PR

**Review by zFernand0 — CHANGES_REQUESTED** [review]
Thanks for updating the readme as well.

I see that license headers have `Zowe Python Client SDK`

For a quick fix,  I suggest doing a find-and-replace with your IDE.

Otherwise, you may need to enhance the existing `scripts/license_header.py` script to account for license header updates 😓

File: CHANGELOG.md
Comment (olamidepeterojo) [reply CHANGELOG.md:145]: okay, your suggestion is good, let me get it done right away

File: CHANGELOG.md
Comment (zFernand0) [inline CHANGELOG.md:144]:
```suggestion
- Fixed the inconsistent use of the SDK name across SDKs in all files by implementing the "find and replace" tool. [#328](https://github.com/zowe/zowe-client-python-sdk/issues/328)
```
Not sure what happened with the bullet point here 😅 
Sorry for being picky 😋

**Review by zFernand0 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md
Comment (zFernand0) [inline CHANGELOG.md:144]:
```suggestion
- Fixed exception handling in session.py [#213](https://github.com/zowe/zowe-client-python-sdk/issues/213)
- Fixed the inconsistent use of the SDK name across SDKs in all files by implementing the "find and replace" tool. [#328](https://github.com/zowe/zowe-client-python-sdk/issues/328)
```

**Review by zFernand0 — CHANGES_REQUESTED** [review]
Because of all the force-push, I'm unable to determine when the changelog entry was added in 1.0.0-dev.11 instead of the Recent Changes. 😢 

Would you mind moving it to the very top?

```markdown
## Recent Changes

### Enhancements

- Turning of logger at the class-constructor level [#316] (https://github.com/zowe/zowe-client-python-sdk/issues/316)

### Bug Fixes

- Fixed the inconsistent use of the SDK name across SDKs in all files by implementing the "find and replace" tool. [#328](https://github.com/zowe/zowe-client-python-sdk/issues/328)

## `1.0.0-dev21`
```

**Review by zFernand0 — APPROVED** [review]
LGTM! 😋

**Comment by olamidepeterojo** [comment]
hey @zFernand0 , seems like you got it done already haha. would you still be needing i make the same changes?

**Comment by zFernand0** [comment]
> hey `@zFernand0` , seems like you got it done already haha. would you still be needing i make the same changes?

Hey @olamidepeterojo,
I felt that I was being super annoying so that's why I tried to help out for once 😅 
Apologies for all the requested changes...

And to answer you question, No.
I think there are no mo changes required/needed in order to get this PR merged in 🙏 

Thanks for all the hard work 🙏 

We just need one more review from the team before we get it merged 🥳

**Comment by olamidepeterojo** [comment]
its no hassle @zFernand0 its nice working with you and its my pleasure.

could you kindly check my PR of issue #336. ive been waiting your review haha.

**Pull request merged**
